partcad/src/partcad/test/cam_additive_solid.py

```
@@ -0,0 +1,43 @@
+
+        # TODO(clairbee): Utilize the data provided in the config
+
+        # TODO(clairbee): Improve and extend the below
+        wrapped = await shape.get_wrapped()
+        fbp = ShapeAnalysis_FreeBoundsProperties(wrapped)
+        fbp.Perform()
+        if fbp.NbFreeBounds() != 0:
+            pc_logging.error("The shape is not solid")
+            return
+
+        pc_logging.debug("Passed test: %s: %s:%s" % (self.name, shape.project_name, shape.name))
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)